File: codemap/indexer/testdata/php/src/Store/PgRepo.php
<?php

namespace Acme\Store;

use Acme\Orders\Order;

class PgRepo implements Repo
{
    public static function table(): string
    {
        return 'orders';
    }

    public function save(Order $order): int
    {
        return $this->insert($order);
    }

    protected function insert(Order $order): int
    {
        return $order->id;
    }
}

File: codemap/indexer/testdata/cs/src/Acme.Store/PgRepo.cs
namespace Acme.Store;

public class PgRepo : IRepo
{
    public static string Table() => "orders";

    public int Count { get; private set; }

    public object Save(object o)
    {
        Count++;
        return Insert(o);
    }

    private object Insert(object o) => o;
}
